TelCoColorCoder.cpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+#include "TelCoColorCoder.h"
+
+const char* TelCoColorCoder::MajorColorNames[] = {
+        "White", "Red", "Black", "Yellow", "Violet"
+};
+
+int numberOfMajorColors =
+sizeof(TelCoColorCoder::MajorColorNames) / sizeof(TelCoColorCoder::MajorColorNames[0]);
+
+const char* TelCoColorCoder::MinorColorNames[] = {
+    "Blue", "Orange", "Green", "Brown", "Slate"
+};
+
+int TelCoColorCoder::numberOfMinorColors =
+sizeof(TelCoColorCoder::MinorColorNames) / sizeof(TelCoColorCoder::MinorColorNames[0]);
+
+
+void TelCoColorCoder::PrintReferenceManual()
+{
+
+    for (int PairNumber = 1; PairNumber <= 25; PairNumber++)
```

Comment:
Avoid hard coding numbers like 25, since we often forget to update them when things change.
In this case, probably the colors will never change. But it's simple to derive 25 by multiplying the number of colors, so better to make it a habit :)